### PR TITLE
Changes for working with odML Editor (version 1.4)

### PR DESCRIPTION
**Pull request by digaru19**

### CODE REVIEW

**Comment by achilleas-k** [comment]
So if I understand correctly, these are event handlers for handling GUI events? Do we want this in the odML core library? Could these not be added to the ui library instead?

**Comment by digaru19** [comment]
@achilleas-k Yeah, removing the GUI handling part makes sense. Implemented it in PR G-Node/odml-ui#50 . Updated this PR's commits too.

File: odml/dtypes.py [hunk 1]
```diff
@@ -28,6 +28,17 @@ class DType(str, Enum):
...
         return self.name
 
+default_values = {
+    'string': '',
+    'text': '',
+    'int': 0,
+    'float': 0.0,
+    'url': '',
+    'datetime': '2015-06-20 10:00:00',
+    'date': '2015-06-20',
```
Comment (achilleas-k) [inline odml/dtypes.py]: Where did these default values come from?
Can we do `now` somehow?

Comment (digaru19) [reply odml/dtypes.py]: You mean for the `datetime`, `date`, and `time` types, right ? 
Yeah, we can implement something like `now` for them. I'll make the changes. Maybe we can have it as a function ?

**Pull request merged**